components/ContractInfo/index.tsx

```diff
@@ -1,107 +1,82 @@
 import type { PolyjuiceContract as PolyjuiceContractProps } from 'components/AccountOverview'
-import { useState, useMemo, useEffect } from 'react'
+import { useState, useEffect } from 'react'
 import { useTranslation } from 'next-i18next'
 import NextLink from 'next/link'
-import { ethers } from 'ethers'
 import OpenInNewIcon from 'assets/icons/open-in-new.svg'
 import ExpandIcon from 'assets/icons/expand.svg'
-import { IS_MAINNET, provider, PCKB_UDT_INFO } from 'utils'
+import { IS_MAINNET, provider, mainnet, testnet } from 'utils'
```

Comment:
Importing `mainnet` or `testnet` could be determined in `utils` in fact because the chain type is determined by an environment variable